### PR TITLE
Fill account fragment with data

### PR DESCRIPTION
**Pull request by skaminskirepo**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -39,8 +39,10 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'com.squareup.okhttp3:logging-interceptor:4.5.0'
     implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'com.jakewharton:butterknife:10.2.1'
```
Comment (GrzegorzFryger) [inline app/build.gradle]: image

**Review by GrzegorzFryger — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/account_fragment.xml [hunk 2]
```diff
@@ -1,14 +1,252 @@
...
-    tools:context=".ui.account.AccountFragment"
-    android:background="@color/colorPrimary">
+    android:background="@color/colorPrimary"
+    android:shrinkColumns="*"
+    android:stretchColumns="*"
+    app:layout_constrainedHeight="false"
+    tools:context=".ui.account.AccountFragment">
 
-    <TextView
+    <TableRow
```
Comment (GrzegorzFryger) [inline app/src/main/res/layout/account_fragment.xml:13]: I would strongly recommend using ConstraintLayout

File: app/src/main/res/layout/account_fragment.xml [hunk 3]
```diff
@@ -1,14 +1,252 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/emptyTextView6"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:gravity="center_horizontal"
+            android:textAlignment="textStart"
```
Comment (GrzegorzFryger) [inline app/src/main/res/layout/account_fragment.xml:23]: the property textStart doesn't exist

File: app/src/main/java/pl/edu/pjatk/pamo/skrawek/ui/account/AccountFragment.java [hunk 4]
```diff
@@ -36,4 +49,17 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         // TODO: Use the ViewModel
     }
 
+    private void fillFragmentWithData(View view) {
+        updateText(view, R.id.fullNameTextView, getProperty(NAME) + " " + getProperty(SURNAME));
+        updateText(view, R.id.cityTextView, getProperty(POSTAL_CODE) + " " + getProperty(CITY));
+        updateText(view, R.id.streetTextView, getProperty(STREET_NUMBER));
+        updateText(view, R.id.phoneTextView, getProperty(PHONE));
+        updateText(view, R.id.accountStatusTextView, getProperty(STATUS));
```
Comment (GrzegorzFryger) [inline app/src/main/java/pl/edu/pjatk/pamo/skrawek/ui/account/AccountFragment.java:57]: Could you transfer the creation of an object to repositry patern?

**Review by GrzegorzFryger — CHANGES_REQUESTED** [review]

File: app/src/main/java/pl/edu/pjatk/pamo/skrawek/ui/account/AccountFragment.java (hunk 4 above)
Comment (skaminskirepo) [reply app/src/main/java/pl/edu/pjatk/pamo/skrawek/ui/account/AccountFragment.java:57]: Within next PR if necessary - this one is already quite big

File: app/build.gradle (hunk 1 above)
Comment (skaminskirepo) [reply app/build.gradle]: Commit with this change removed

File: app/src/main/res/layout/account_fragment.xml (hunk 2 above)
Comment (skaminskirepo) [reply app/src/main/res/layout/account_fragment.xml:13]: Why?

File: app/src/main/res/layout/account_fragment.xml (hunk 3 above)
Comment (skaminskirepo) [reply app/src/main/res/layout/account_fragment.xml:23]: Fixed

**Pull request merged**